src/core/dune

```diff
@@ -19,6 +19,9 @@
                   ((action (run %{bin:cppo} -D "VERSION %{read-lines:version}" %{input-file})) opamVersion)))
   (wrapped     false))
 
+(rule
+  (copy opamStubsTypes.ml opamStubsTypes.mli))
+
```

Comment:
```suggestion
                  ((action (run %{bin:cppo} -D "VERSION %{read-lines:version}" %{input-file})) opamVersion)))
  (modules_without_implementation opamStubsTypes)
  (wrapped     false))

```

Reply:
I don't think that works - this is a module without an interface?

Reply:
ah right, then we just need to rename the files at the same time `.ml --> .mli`

Reply:
No - that's why I didn't... that changes the library and I didn't have the courage of my convictions to do that just to silence a warning 🙂

Reply:
(cf. the breakages in opam-file-format prior to 2.1.0!)